src/app/tabs/registrar/containers/RevealContainer.js

```diff
@@ -20,14 +20,14 @@ const mapStateToProps = state => ({
 
 const mapDispatchToProps = dispatch => ({
   // eslint-disable-next-line max-len
```

Comment:
Can we remove this two eslint disables? We should follow linting :)

Reply:
fixed.